src/CryptState.cpp

```diff
@@ -32,21 +32,28 @@
 #include <openssl/rand.h>
 
 CryptState::CryptState() {
+	uiGood = uiLate = uiLost = uiResync = 0;
+	uiRemoteGood = uiRemoteLate = uiRemoteLost = uiRemoteResync = 0;
+	bInit = false;
+}
```

Comment:
member initialization should be done via cpp11's "non-static member initializer" feature - see https://arne-mertz.de/2015/08/new-c-features-default-initializers-for-member-variables/ (but please use the syntax involving the `=`).